mne/encoding/model.py

```diff
@@ -0,0 +1,329 @@
+import numpy as np
+from sklearn.linear_model import Ridge
+from sklearn.pipeline import Pipeline
```

Comment:
use nested imports for sklearn. Don't make sklearn a hard dependency.

it should help travis.